src/components/Footer.astro

```diff
@@ -208,3 +145,38 @@ import { Icon } from 'astro-icon';
         </div>
     </div>
 </footer>
+
+<style lang="scss">
+    @import 'bootstrap/scss/bootstrap.scss';
+    @import 'bootstrap-print-css/css/bootstrap-print.min';
+    @import '../styles/_variables.scss';
```

Comment:
My guess is that we will use this three quite often together. Should we have a `base.scss` which just imports these?

Reply:
Yup, good call 👍🏻